doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -122,8 +122,15 @@ For lists of protocol-specific changes, see `Protocols`_.
 Bluetooth samples
 -----------------
 
-|no_changes_yet_note|
+* :ref:`peripheral_hr_coded` sample:
+
+  * Added configuration for the nRF5340 target.
+  * Fixed advertising start on the nRF5340 target with the Zephyr LL controller.
```

Comment:
I believe this is fine as is, but do you think the fix could be explained a bit more? For example something like:
"Fixed the behavior of the advertising start process on..."
I don't really know whether that would make this any clearer. Up to you to decide.
Same with the next item, if you think this makes any sense.

Reply:
I added a sentence which explains what was an issue here I hope that it has more sense now